src/config/dynamic-theme-fixes.config

```diff
@@ -3820,13 +3820,10 @@ CSS
 
 linkedin.com
 
-INVERT
-.artdeco-button
-.jobs-search-box__input-icon
-.artdeco-pill
-.artdeco-tabs .artdeco-tab
-
 CSS
+:root {
+    --color-text-low-emphasis: $(gray);
```

Comment:
Did you mean `--color-text-low-emphasis: ${gray} !important;`?

Reply:
yup... still checking how it will work with all places around linkedin

Reply:
normaly there was some dark color set as variable. If it will make some text too dark somewhere then I will try to make the original one included here.

Reply:
It will not work with round brackets.